libs/features/mas/docs/merch-card.html

```diff
@@ -10,10 +10,17 @@
     <link rel="stylesheet" href="https://use.typekit.net/hah7vzn.css">
   
   <script>
+      const ENVS = {
+        qa: 'qa-odin',
+        stage: 'stage-odin',
+        prod: 'odin',
+      };
```

Comment:
there is same change in @npeltier  PR, we should either remove it here or in his PR 
https://github.com/adobecom/milo/pull/3377/files#diff-3521e479d9bca2a18c02ab92f38233c4b7370eab1e0888dec951ab091426375dR6

Reply:
I'll remove mine, his is better.